temporalio/workflow.py

```diff
@@ -584,6 +630,10 @@ def workflow_set_update_handler(
     ) -> None:
         ...
 
+    @abstractmethod
+    def workflow_all_handlers_finished(self) -> bool:
```

Comment:
A bit pedantic, but I think there's value in keeping these `workflow_` calls in alphabetical order here and in the impl